### PR TITLE
add generic V&V reporting

### PR DESCRIPTION
**Pull request by Rikkola**
https://issues.redhat.com/browse/DROOLS-5203

https://github.com/kiegroup/drools/pull/2828
https://github.com/kiegroup/kie-wb-common/pull/3250
https://github.com/kiegroup/drools-wb/pull/1342

### CODE REVIEW

**Comment by Rikkola** [comment]
[ERROR] Internal error: java.lang.RuntimeException: org.eclipse.equinox.p2.core.ProvisionException: Unable to read repository at http://download.jboss.org/jbosstools/updates/requirements/m2eclipse-tycho/0.8.1.201704211436/artifacts.jar. connect timed out -> [Help 1]

Jenkins please restest

File: kie-wb-common-dmn/kie-wb-common-dmn-webapp-standalone/pom.xml [hunk 1]
```diff
@@ -914,6 +929,8 @@
                 <compileSourcesArtifact>org.kie.workbench.services:kie-wb-common-datamodel-api</compileSourcesArtifact>
                 <compileSourcesArtifact>org.kie.workbench.services:kie-wb-common-services-api</compileSourcesArtifact>
                 <compileSourcesArtifact>org.kie.workbench:kie-wb-common-kogito-client</compileSourcesArtifact>
+<!--                <compileSourcesArtifact>org.drools:drools-verifier-api</compileSourcesArtifact>-->
+<!--                <compileSourcesArtifact>org.drools:drools-verifier-core</compileSourcesArtifact>-->
```
Comment (Rikkola) [inline kie-wb-common-dmn/kie-wb-common-dmn-webapp-standalone/pom.xml:933]: Need to rush and remove this commented code!

**Comment by manstis** [comment]
**_null error_**

- Add InputData
- Set data-type to `string`
- Add Decision
- Set expression to Decision Table
- Do nothing else
- Validate

Screenshot from 2020-04-08 10-29-07


**_Index error_**

- Add InputData
- Set data-type to `string`
- Add Decision
- Set expression to Decision Table
- Set output column datatype to `number`
- Set output value cell to a number
- Validate

Screenshot from 2020-04-08 10-27-47

**Comment by jomarko** [comment]
@Rikkola I will postpone until you have look on @manstis comments, seems quite  blocking

**Comment by Rikkola** [comment]
@manstis @jomarko it looks like this issue was already there. I'll make ticket.

**Comment by manstis** [comment]
@Rikkola OK, I can confirm the `null` case exists with `master`

Screenshot from 2020-04-08 15-46-42

However the second case (where this PR shows a message relating to "index") is better reported by `master`. Can the message produced by this PR be consistent with `master` so Users have some idea what is going on?

Screenshot from 2020-04-08 15-46-55

**Comment by manstis** [comment]
@Rikkola I tried the following simple case in both `master` and this PR.

**_Subsumption correctly reported by master_** 

Screenshot from 2020-04-08 15-49-17

**_Index error reported by this PR_**

Screenshot from 2020-04-08 15-58-56

**Comment by manstis** [comment]
@Rikkola Can I please also ask, to aid testing, can you provide a table showing the different _checks_ and where each is _implemented_ for example:

Check     | old kie-dmn | new drools-vv
------------|-------------|--------------              
Impossible range |      x      |              
Subsumption |       x      |              
Gap analysis |       x      |       

etc

**Pull request closed without merging**